### PR TITLE
Add cypress bucket-class test

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Migrate cypress bucket-class test from ocs-console 
Signed-off-by: Timothy Asir Jeyasingh <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@bipuladh , @afreen23, @vbnrh  Please review

File: cypress/views/common.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+export const commonFlows = {
+  navigateToOCS: (isOperatorsOpen = false) => {
```
[the remark this answers is not in the document]
Comment (vbnrh) [reply cypress/views/common.ts:2]: Please change to go to the ODF page and then bucket class

**Review by vbnrh — CHANGES_REQUESTED** [review]
Please post screenshots of the tests too in the description

File: cypress/views/common.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (TimothyAsirJeyasing) [reply cypress/views/common.ts:2]: Sure, will do it

File: cypress/views/bc.ts [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+  cy.log('Deleting bucket class');
+  cy.byLegacyTestID('actions-menu-button').click();
+  cy.byTestActionID('Delete Bucket Class').click();
+  cy.byTestID('confirm-action').click();
+  cy.byTestID('item-create').should('be.visible');
+};
+
+export const visitBucketClassPage = () => {
+  cy.visit('/');
+  commonFlows.navigateToODF();
```
Comment (vbnrh) [inline cypress/views/bc.ts:212]:
```suggestion
 ODFCommon.visitStorageDashboard();
```

File: cypress/cypress.json [hunk 3]
```diff
@@ -1,21 +1,4 @@
...
-  },
-  "screenshotOnRunFailure": true,
-  "supportFile": "cypress/support.ts",
-  "pluginsFile": "cypress/plugin.js",
-  "fixturesFolder": false,
-  "defaultCommandTimeout": 30000,
-  "retries": {
-    "runMode": 1,
-    "openMode": 0
-  }
```
Comment (vbnrh) [inline cypress/cypress.json:20]: Why are we removing all these ?

Comment (TimothyAsirJeyasing) [reply cypress/cypress.json:20]: I found these attributes returns error during the test setup. Will try find the main cause and update accordingly

Comment (TimothyAsirJeyasing) [reply cypress/cypress.json]: I have taken these attribute in my latest push, Thank you

**Review by bipuladh — APPROVED** [review]
/lgtm

**Pull request merged**